src/pages/BookPage/style.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../assets/main.css';
+.blue-text {
+  color: #2f80ed;
```

Comment:
each color should be a variable and try to ask your teammates about what color did they used so you will be on the same page